packages/lib/src/ui/blocks/Gallery/galleryDef.tsx

```diff
@@ -0,0 +1,45 @@
+import { BlockTemplates } from "../enums"
+import { Block, BlockDef } from "../blocks"
+import { BlockFields, withBlockSchema } from "../../components/Block/Block"
+import * as yup from "yup"
+import { Image, imageDef, imageSchema } from "../../../types/Image"
+
+interface GalleryImageProps {
+  image: Image
+}
+
+export interface GalleryBlock extends Block {
+  template: BlockTemplates.Gallery
+  fields: GalleryProps
+}
+
+export interface GalleryProps extends BlockFields {
+  content: GalleryImageProps[]
```

Comment:
Není to zbytečně zanořené?
Co takhle:
```suggestion
images: Image[]
```